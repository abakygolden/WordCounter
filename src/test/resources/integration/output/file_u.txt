UDHVILEDE 2
UDSAT 1
ULTRICES 1
ULTRICIES 1
UN 4
UNA 1
UNI 1
UNIDOS 1
UNOS 1
URNA 2
US 3
USTEDES 1
UT 2