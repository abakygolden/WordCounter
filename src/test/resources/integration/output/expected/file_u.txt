UGLI 4
UPSIDE 4